charts/kaito/workspace/templates/clusterrole.yaml

```diff
@@ -39,7 +39,10 @@ rules:
   - apiGroups: ["admissionregistration.k8s.io"]
     resources: ["validatingwebhookconfigurations"]
     verbs: ["get","list","watch"]
+  - apiGroups: [ "batch" ]
+    resources: [ "jobs" ]
+    verbs: [ "get", "list", "watch", "create", "delete", "update", "patch" ]
   - apiGroups: ["admissionregistration.k8s.io"]
     resources: ["validatingwebhookconfigurations"]
     verbs: ["update"]
-    resourceNames: ["validation.workspace.kaito.sh"]
+    resourceNames: ["validation.workspace.kaito.sh"]
```

Comment:
nit: add new line. [why?](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)